### PR TITLE
feat: mask phone numbers in no quota and redeem success screens

### PR DESCRIPTION
**Pull request by szenius**
This PR
* Adds new util function to display identifier inputs, which also masks identifier inputs when they are phone numbers
* Above util function is now used for no quota and redeemed screens

### CODE REVIEW

File: src/utils/getIdentifierInputDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { IdentifierInput } from "../types";
+
+const processIdentifierInputValue = (
+  identifierInput: IdentifierInput
+): string => {
+  return identifierInput.textInputType === "PHONE_NUMBER"
+    ? `****${identifierInput.value.slice(identifierInput.value.length - 4)}`
```
Comment (sebastianquek) [inline src/utils/getIdentifierInputDisplay.tsx:7]: Perhaps can use `padStart`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Comment (szenius) [reply src/utils/getIdentifierInputDisplay.tsx:7]: From discussion with Pat, we can display fixed 4 *s at the start. Let me ask if she prefers to pad as per original length, if yes I'll change to this. If not, I think might be more straightforward to do 4 *s?

**Comment by szenius** [comment]
@ilhaoni Good catch, made the fix!

File: src/utils/getIdentifierInputDisplay.tsx (hunk 1 above)
Comment (sebastianquek) [reply src/utils/getIdentifierInputDisplay.tsx:7]: Oh I was thinking to move this to some generic `maskText` function that can be used for anything, including NRIC, so just need to configure how many characters to show.

Comment (szenius) [reply src/utils/getIdentifierInputDisplay.tsx]: Pat said prefers to retain length, so I changed to padStart. Thanks for the suggestion 🙌

**Pull request merged**